.github/renovate.json5

```diff
@@ -54,72 +54,33 @@
         "!/^com.mysql/",
         "!/^org.jboss.resteasy/",
         "!/^org.apache.groovy/"
-      ]
+      ],
+      "enabled": false
     },
-    // Patch
     {
       //"automerge": true // auto-merge will be activated in a second step
-      "matchPackageNames": [
-        "/^commons-fileupload/",
-        "/^org.apache.commons:commons-lang3/",
-        "/^net.minidev:json-smart/",
-        "/^com.jayway.jsonpath:json-path/",
-        "/^org.apache.httpcomponents.client5/",
-        "/^com.fasterxml.uuid/",
-        "/^com.fasterxml.jackson/",
-        "/^org.camunda.feel/",
-        "/^io.quarkus/",
-        "/^ch.qos.logback/",
-        "/^org.graalvm/",
-        "/^com.google.code.gson/",
-        "/^org.postgresql/",
-        "/^com.h2database/",
-        "/^com.mysql/",
-        "/^org.jboss.resteasy/",
-        "/^org.apache.groovy/"
-      ],
-      "updateTypes": [
+      "matchUpdateTypes": [
         "patch"
       ],
       "addLabels": [
         "automerge",
         "ci:all"
       ],
-      "branchPrefix": "{{{baseBranch}}}-"
+      "branchPrefix": "",
+      "additionalBranchPrefix": "{{{baseBranch}}}-"
     },
-    // major, minor
     {
-      "matchPackageNames": [
-        "/^org.springframework/",
-        "/^org.apache.tomcat/",
-        "/^commons-fileupload/",
-        "/^org.apache.commons:commons-lang3/",
-        "/^net.minidev:json-smart/",
-        "/^com.jayway.jsonpath:json-path/",
-        "/^org.apache.httpcomponents.client5/",
-        "/^com.fasterxml.uuid/",
-        "/^com.fasterxml.jackson/",
-        "/^org.camunda.feel/",
-        "/^io.quarkus/",
-        "/^ch.qos.logback/",
-        "/^org.graalvm/",
-        "/^com.google.code.gson/",
-        "/^org.postgresql/",
-        "/^com.h2database/",
-        "/^com.mysql/",
-        "/^org.jboss.resteasy/",
-        "/^org.apache.groovy/"
-      ],
-      "updateTypes": [
+      "matchUpdateTypes": [
         "major",
         "minor"
       ],
       "addLabels": [
         "ci:all"
       ],
-      "branchPrefix": "{{{baseBranch}}}-",
+      "branchPrefix": "",
+      "additionalBranchPrefix": "{{{baseBranch}}}-",
```

Comment:
❓ I was intrigued by these and checked the docs which say `branchPrefix must be configured at the root of the configuration (e.g. not within any package rule)` - I think these lines are still withing `packageRules`. Should it be moved up to the root? Same for lines 69-70. 
Can also just try this way first up to you, the docs arent entirely clear to me 🤷‍♀️

Reply:
Now branches end up as `7.23-7.23-version.tomcat` :(

Reply:
Will try with 
```
  "branchPrefix": "{{{baseBranch}}}-",
```

Reply:
Doesn't work. Will try with:
```
  "branchPrefix": "{{{baseBranch}}}-",
  "additionalBranchPrefix": "",
```

Reply:
This works. See: https://github.com/camunda/camunda-bpm-platform-maintenance/pull/1498

Reply:
The [docs](https://docs.renovatebot.com/configuration-options/#branchname) are a little passive aggressive about that one: 
```
BranchName 
If you truly need to configure this then it probably means either: You are hopefully mistaken, and there's a better approach you should use
```
😆